dolphinscheduler-task-plugin/dolphinscheduler-task-mlflow/src/main/java/org/apache/dolphinscheduler/plugin/task/mlflow/MlflowConstants.java

```diff
@@ -86,9 +88,17 @@ private MlflowConstants() {
 
     public static final String MLFLOW_BUILD_DOCKER = "mlflow models build-docker -m %s -n %s --enable-mlserver";
 
-
     public static final String DOCKER_RREMOVE_CONTAINER = "docker rm -f %s";
 
     public static final String DOCKER_RUN = "docker run --name=%s -p=%s:8080 %s";
 
+    public static final String DOCKER_COMPOSE_RUN = "docker compose up -d";
+
+    public static final String DOCKER_COMPOSE_CANCEL = "docker compose down";
+
+    public static final String SET_DOCKER_COMPOSE_ENV = "export imageName=%s\n" +
+            "export deployPort=%s\n" +
+            "export cpuLimit=%s\n" +
+            "export memoryLimit=%s";
```

Comment:
Generally environment names should be all capitalized, and please add reasonable prefix (like `DS_TASK_MLFLOW_`) to avoid possible conflicts

Reply:
Thanks for your suggestion. I've fixed it.